README.md

```diff
@@ -0,0 +1,88 @@
+microarchitecture. It is primarily used to evaluate new hardware designs,
+system software changes, and compile-time and run-time system optimizations.
+
+The main website can be found at <http://www.gem5.org>.
+
+## Getting started
+
+A good starting point is <http://www.gem5.org/about>, and for
+more information about building the simulator and getting started
+please see <http://www.gem5.org/documentation> and
+<http://www.gem5.org/documentation/learning_gem5/introduction>.
+
+## Building gem5
+
+To build gem5, you will need the following software: g++ or clang,
+Python (gem5 links in the Python interpreter), SCons, zlib, m4, and lastly
+protobuf if you want trace capture and playback support. Please see
+<http://www.gem5.org/documentation/general_docs/building> for more details
+concerning the minimum versions of these tools.
+
+Once you have all dependencies resolved, type
+`scons build/<CONFIG>/gem5.opt` where `<CONFIG>` is one of the options in
+build_opts like `ARM`, `NULL`, `MIPS`, `POWER`, `SPARC`, `X86`,
+`Garnet_standalone`, etc. This will build an optimized version of the gem5
+binary (gem5.opt) with the the specified configuration. See
+http://www.gem5.org/documentation/general_docs/building for more details and
+options.
+
+## The Source Tree
+
+The main source tree includes these subdirectories:
+
+* build_opts: pre-made default configurations for gem5
+* build_tools: tools used internally by gem5's build process.
+* configs: example simulation configuration scripts
+* ext: less-common external packages needed to build gem5
+* include: include files for use in other programs
+* site_scons: modular components of the build system
+* src: source code of the gem5 simulator
+* system: source for some optional system software for simulated systems
+* tests: regression tests
+* util: useful utility programs and files
+
+## gem5 Resources
+
+To run full-system simulations, you may need compiled system firmware, kernel
+binaries and one or more disk images, depending on gem5's configuration and
+what type of workload you're trying to run. Many of these resources can be
+obtained from <https://resources.gem5.org>.
+
+More information on gem5 Resources can be found at
+<https://www.gem5.org/documentation/general_docs/gem5_resources/>.
+
+## Getting Help, Reporting bugs, and Requesting Features
+
+We provide a variety of channels for users and developers to get help, report
+bugs, requests features, or engage in community discussions. Below
+are a few of the most common we recommend using.
+
+* **gem5-users@gem5.org**: A mailing list for users of gem5 to ask questions
+or start discussions. To join the mailing list please visit
+<https://www.gem5.org/mailing_lists>.
+* **gem5-dev@gem5.org**: A mailing list for developers of gem5 to ask questions
+or start discussions. To join the mailing list please visit
+<https://www.gem5.org/mailing_lists>.
+* **GitHub Discussions**: A GitHub Discussions page. This can be used to start
```

Comment:
Reorder with this at the top? Do we want to encourage github first and mailing list last?

Similarly, maybe we should move slack up?

Reply:
I've reordered this.